buildings/gui/alter_building_relationships.py

```diff
@@ -1408,3 +1459,120 @@ def select_row_in_tbl_added(self, id_bulk):
             if int(tbl.item(row, 0).text()) == id_bulk:
                 tbl.selectRow(row)
                 tbl.scrollToItem(tbl.item(row, 0))
+
+    def canvas_add_outline(self):
+        for val in [str(layer.id()) for layer in iface.legendInterface().layers()]:
+            if 'existing_subset_extracts' in val:
+                self.lyr_existing.removeSelection()
```

Comment:
`self.lyr_existing` and `self.lyr_bulk_load` can be used without the 'for loop and if statement' as they should always be in the layer panel when alter-relationship-frame is on. The same for canvas_edit_geometry/attribute and edit_cancel_clicked